### ISSUE TITLE
I'm having error when i run my flutter app in debug mode

### ISSUE THREAD

**Issue by HigorFerreira** [flutter/flutter#39828]
Hello, when i run ```flutter run```, console shows me a lot of error messages that apparently does not affect my app running:
```
D/EGL_emulation(31864): eglMakeCurrent: 0xa5685360: ver 2 0 (tinfo 0xa5683230)
E/flutter (31864): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: Exception: operation failed
E/flutter (31864): #0      ShaderWarmUp.execute (package:flutter/src/painting/shader_warm_up.dart:90:5)
E/flutter (31864): <asynchronous suspension>                            
E/flutter (31864): #1      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding.initInstances (package:flutter/src/painting/binding.dart:25:20)
E/flutter (31864): #2      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding.initInstances (package:flutter/src/semantics/binding.dart:22:11)
E/flutter (31864): #3      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding.initInstances (package:flutter/src/rendering/binding.dart:29:11)
E/flutter (31864): #4      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.initInstances (package:flutter/src/widgets/binding.dart:252:11)
E/flutter (31864): #5      new BindingBase (package:flutter/src/foundation/binding.dart:56:5)
E/flutter (31864): #6      new _WidgetsFlutterBinding&BindingBase&GestureBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #7      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #8      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #9      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #10     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #11     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #12     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #13     new WidgetsFlutterBinding (package:flutter/src/widgets/binding.dart)
E/flutter (31864): #14     WidgetsFlutterBinding.ensureInitialized (package:flutter/src/widgets/binding.dart:994:7)
E/flutter (31864): #15     runApp (package:flutter/src/widgets/binding.dart:785:25)
E/flutter (31864): #16     main (package:animations/main.dart:4:16)     
E/flutter (31864): #17     _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:216:25)
E/flutter (31864): #18     _rootRun (dart:async/zone.dart:1124:13)      
E/flutter (31864): #19     _CustomZone.run (dart:async/zone.dart:1021:19)
E/flutter (31864): #20     _runZoned (dart:async/zone.dart:1516:10)     
E/flutter (31864): #21     runZoned (dart:async/zone.dart:1500:12)      
E/flutter (31864): #22     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:208:5)
E/flutter (31864): #23     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:301:19)
E/flutter (31864): #24     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
E/flutter (31864):                                                      
Syncing files to device Android SDK built for x86...                    
 3.387ms (!)   
```
I tried to run flutter upgrade to maybe solve some issue, because that messages appear after a flutter upgrade i ran sometime ago but nothing works. I don't know i'm confident that this messages will cause some problem to me later.
Can anyone help me to disappear with that messages?

**Comment by BondarenkoStas** [comment]
@HigorFerreira Please provide your ```flutter doctor -v``` output.

**Comment by HigorFerreira** [comment]
@BondarenkoStas , here is it:

[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Linux, locale pt_BR.UTF-8)
    • Flutter version 1.7.8+hotfix.4 at /home/higor/flutter/flutter_linux_v1.5.4-hotfix.2-stable/flutter
    • Framework revision 20e59316b8 (7 weeks ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /home/higor/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = /home/higor/Android/Sdk
    • Java binary at: /home/higor/Downloads/android-studio-ide-181.5014246-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Android Studio (version 3.2)
    • Android Studio at /home/higor/Downloads/android-studio-ide-181.5014246-linux/android-studio
    • Flutter plugin version 31.3.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.36.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 7.0 (API 24) (emulator)

• No issues found!

**Comment by HigorFerreira** [comment]
The problem is probably cause by flutter channel, I had not observed that my flutter channel it was in the dev channel, to prevent error, keep your channel in stable

**Comment by HigorFerreira** [comment]
The problem isn't with the flutter channel, this problem are occurring in the stable channel :(

**Comment by remoteportal** [comment]
Yeah, this happens EVERY time I start my app.  It's so annoying.

**Comment by Fernandomr88** [comment]
Same problem here.
![image](https://user-images.githubusercontent.com/14304128/68617664-76a79d80-04a6-11ea-854c-3996dbaa6ac9.png)

Nothing runs before that.

**Comment by Fernandomr88** [comment]
I'm switched to ```flutter channel v1.9.1-hotfixes``` and it's now working

**Comment by remoteportal** [comment]
When are they going to fix this?  You can't write even a "hello, world" app without a huge ShaderWarmUp exception.

**Comment by bleroux** [comment]
I had the same problem on 'Channel master, v1.10.17-pre.52'. Now i'm on 'Channel master, v1.12.2-pre.51' and don't have this problem.
@remoteportal Could you try with this build ? It not thank you to provide your flutter doctor -v output.

**Comment by qinliwei123** [comment]
How to deal with this problem

**Comment by tvolkert** [comment]
@liyuqian for the shader warm-up

@Fernandomr88 your issue looks unrelated to the reported issue here - please file a new issue with details on how to reproduce.

**Comment by tvolkert** [comment]
@HigorFerreira can you post details on how to reproduce, including sample code?

**Comment by liyuqian** [comment]
According to https://github.com/flutter/flutter/issues/39828#issuecomment-554725592, this issue seems to have been fixed in the master branch. @HigorFerreira @qinliwei123 : can you please try to use the newest master branch Flutter and see if the error message is still there? If there are still errors, can you please post the result of `flutter doctor -v`?

**Comment by synapp009** [comment]
Problem still persists with actual master channel.

```
Flutter (Channel master, v1.15.3-pre.37, on Mac OS X 10.14.6 18G95, locale de-DE)
    • Flutter version 1.15.3-pre.37 at ****/flutter
    • Framework revision 6dc3bfaa98 (vor 3 Tagen), 2020-02-07 18:28:02 -0800
    • Engine revision 6158f03ef5
    • Dart version 2.8.0 (build 2.8.0-dev.8.0 514a8d4c84)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /***y/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[!] IntelliJ IDEA Community Edition (version 2019.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.42.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.8.1

[✓] Connected device (4 available)
    • Custom            • 192.168.56.101:5555                  • android-x86    • Android 9 (API
      28)
    • iPhone 11 Pro Max • 9ED807EB-5D04-46ED-B807-11B9E9424FC0 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-13-1 (simulator)
    • Chrome            • chrome                               • web-javascript • Google Chrome
      79.0.3945.130
    • Web Server        • web-server  
```

**Comment by liyuqian** [comment]
@synapp009 : I tried to reproduce it using the version 6dc3bfaa98 in your `flutter doctor -v` but can't see the error message. What device did you use? I used the Android emulator suggested by https://github.com/flutter/flutter/issues/39828#issue-489274689.

Below are my setup and my `flutter run` output:
```
liyuqian-macbookpro2:flutter liyuqian$ flutter doctor -v
[✓] Flutter (Channel unknown, v1.15.3-pre.37, on Mac OS X 10.15.3 19D76, locale en-US)
    • Flutter version 1.15.3-pre.37 at /Users/liyuqian/flutter/flutter
    • Framework revision 6dc3bfaa98 (9 days ago), 2020-02-07 18:28:02 -0800
    • Engine revision 6158f03ef5
    • Dart version 2.8.0 (build 2.8.0-dev.8.0 514a8d4c84)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/liyuqian/Library/Android/sdk/
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/liyuqian/Library/Android/sdk/
    • ANDROID_SDK_ROOT = /Users/liyuqian/Library/Android/sdk/

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio with Blaze.app/Contents
    • Flutter plugin version 43.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Connected device (3 available)
    • AOSP on IA Emulator • emulator-5554 • android-x86    • Android 9 (API 28) (emulator)
    • Chrome              • chrome        • web-javascript • Google Chrome 80.0.3987.106
    • Web Server          • web-server    • web-javascript • Flutter Tools
```

```
liyuqian-macbookpro2:hello liyuqian$ flutter run
Using hardware rendering with device AOSP on IA Emulator. If you get graphics artifacts, consider enabling software rendering with "--enable-software-rendering".
Launching lib/main.dart on AOSP on IA Emulator in debug mode...
Running Gradle task 'assembleDebug'...
Running Gradle task 'assembleDebug'... Done                         2.3s
✓ Built build/app/outputs/apk/debug/app-debug.apk.
D/EGL_emulation( 6806): eglMakeCurrent: 0xe7205240: ver 3 0 (tinfo 0xe7203280)
D/eglCodecCommon( 6806): setVertexArrayObject: set vao to 0 (0) 1 0
Syncing files to device AOSP on IA Emulator...
 5,745ms (!)

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
```

**Comment by synapp009** [comment]
Sorry for responding in delay.
I'm using Genymotion with Android 9.0 - API 28

**Comment by liyuqian** [comment]
Genymotion doesn't have a free version for me to try out... @synapp009 : can you please try the emulator from Android Studio and see if the error disappears?

Meanwhile, CC @cbracken @tvolkert to see if we should buy [Genymotion](https://www.genymotion.com/pricing-desktop/) for testing.

**Comment by synapp009** [comment]
Ok. I've tried it with the qemu_system-x86_64 and it worked without problems.
(Android Studio emulator would take hours to set up because I'm working in a public library without access to a good internet connection, sorry..)

There should be a free (personal) version of genymotion.

**Comment by HigorFerreira** [comment]
The error persists here too

**Comment by HigorFerreira** [comment]
> According to [#39828 (comment)](https://github.com/flutter/flutter/issues/39828#issuecomment-554725592), this issue seems to have been fixed in the master branch. @HigorFerreira @qinliwei123 : can you please try to use the newest master branch Flutter and see if the error message is still there? If there are still errors, can you please post the result of `flutter doctor -v`?

The error persists

**Comment by ifin2000** [comment]
I only change package name (com.example.***) at /android/app/src/main/kotlin/MainActivity.kt as the same with applicationID at /android/app/build.gradle, and now it's running...

**Comment by remoteportal** [comment]
> I had the same problem on 'Channel master, v1.10.17-pre.52'. Now i'm on 'Channel master, v1.12.2-pre.51' and don't have this problem.
> @remoteportal Could you try with this build ? It not thank you to provide your flutter doctor -v output.

[√] Flutter (Channel stable, v1.12.13+hotfix.8, on Microsoft Windows [Version 6.1.7601], locale en-US)
    • Flutter version 1.12.13+hotfix.8 at E:\Src\flutter
    • Framework revision 0b8abb4724 (5 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Users\pete\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
    • All Android licenses accepted.

[√] Android Studio (version 3.5)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)

[√] Connected device (2 available)
    • Mi 9T                     • 844fb80a      • android-arm64 • Android 9 (API 28)
    • Android SDK built for x86 • emulator-5554 • android-x86   • Android 8.1.0 (API 27) (emulator)

• No issues found!

**Comment by CaiJingLong** [comment]
The problem also happened on my genymotion.

<details>

```log
E/flutter ( 5868): #7      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #8      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #9      new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #10     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #11     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #12     new _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #13     new WidgetsFlutterBinding (package:flutter/src/widgets/binding.dart)
E/flutter ( 5868): #14     WidgetsFlutterBinding.ensureInitialized 
package:flutter/…/widgets/binding.dart:1083
E/flutter ( 5868): #15     runApp 
package:flutter/…/widgets/binding.dart:868
E/flutter ( 5868): #16     main 
package:image_scanner_example/main.dart:10
E/flutter ( 5868): #17     _runMainZoned.<anonymous closure>.<anonymous closure>  (dart:ui/hooks.dart:239:25)
E/flutter ( 5868): #18     _rootRun  (dart:async/zone.dart:1126:13)
E/flutter ( 5868): #19     _CustomZone.run  (dart:async/zone.dart:1023:19)
E/flutter ( 5868): #20     _runZoned  (dart:async/zone.dart:1518:10)
E/flutter ( 5868): #21     runZoned  (dart:async/zone.dart:1502:12)
E/flutter ( 5868): #22     _runMainZoned.<anonymous closure>  (dart:ui/hooks.dart:231:5)
E/flutter ( 5868): #23     _startIsolate.<anonymous closure>  (dart:isolate-patch/isolate_patch.dart:307:19)
E/flutter ( 5868): #24     _RawReceivePortImpl._handleMessage  (dart:isolate-patch/isolate_patch.dart:174:12)
E/flutter ( 5868):

```


</details>

**Comment by liyuqian** [comment]
I recently tried Genymotion `Version 3.1.0 | Revision 20200403-ca22008` with Flutter version 8c0b43bda4e5ff29d399f06ce1c0e1027268d6b5 and 0b8abb4724aa590dd0f429683339b1e045a1594d (v1.12.13+hotfix.8). It seems that the error messages are gone in the new version 8c0b43bda4e5ff29d399f06ce1c0e1027268d6b5.

Can someone else please confirm if this issue is already fixed in the master branch with Genymotion?

Here are  more details:

## 8c0b43bda4e5ff29d399f06ce1c0e1027268d6b5  logs
<details>

```
PS E:\src\flutter\flutter\dev\integration_tests\flutter_gallery> flutter run
Launching lib\main.dart on Google Pixel 2 in debug mode...
Running Gradle task 'assembleDebug'...
Running Gradle task 'assembleDebug'... Done                         4.3s
√ Built build\app\outputs\apk\debug\app-debug.apk.
Syncing files to device Google Pixel 2...                        1,668ms

Flutter run key commands.
r Hot reload.
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on Google Pixel 2 is available at: http://127.0.0.1:61451/RegEi1BSU9s=/

Application finished.

PS E:\src\flutter\flutter\dev\integration_tests\flutter_gallery> flutter doctor -v
[√] Flutter (Channel master, v1.16.2-pre.41, on Microsoft Windows [Version 10.0.18362.720], locale en-US)
    • Flutter version 1.16.2-pre.41 at E:\src\flutter\flutter
    • Framework revision 8c0b43bda4 (3 weeks ago), 2020-03-20 19:01:01 -0400
    • Engine revision fdf4f7883f
    • Dart version 2.8.0 (build 2.8.0-dev.16.0 0296286c03)


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\Yuqian\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] VS Code (version 1.43.2)
    • VS Code at C:\Users\Yuqian\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.8.1

[√] Connected device (3 available)
    • Google Pixel 2 • 192.168.168.101:5555 • android-x86    • Android 9 (API 28)
    • Chrome         • chrome               • web-javascript • Google Chrome 80.0.3987.163
    • Web Server     • web-server           • web-javascript • Flutter Tools
```
</details>

## 0b8abb4724aa590dd0f429683339b1e045a1594d logs
<details>

```
PS E:\src\flutter\flutter\examples\flutter_gallery> flutter run
Launching lib\main.dart on Google Pixel 2 in debug mode...
Running Gradle task 'assembleDebug'...
Running Gradle task 'assembleDebug'... Done                         3.8s
√ Built build\app\outputs\apk\debug\app-debug.apk.
D/FlutterActivity( 2020): Using the launch theme as normal theme.
D/FlutterActivityAndFragmentDelegate( 2020): Setting up FlutterEngine.
D/FlutterActivityAndFragmentDelegate( 2020): No preferred FlutterEngine was provided. Creating a new FlutterEngine for this FlutterFragment.
D/FlutterActivityAndFragmentDelegate( 2020): Attaching FlutterEngine to the Activity that owns this Fragment.
D/FlutterView( 2020): Attaching to a FlutterEngine: io.flutter.embedding.engine.FlutterEngine@dee8b42
D/FlutterActivityAndFragmentDelegate( 2020): Executing Dart entrypoint: main, and sending initial route: /
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
D/skia    ( 2020): Shader compilation error
D/skia    ( 2020): ------------------------
D/skia    ( 2020): Errors:
D/skia    ( 2020):
Syncing files to device Google Pixel 2...
 6,032ms (!)

�  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
An Observatory debugger and profiler on Google Pixel 2 is available at: http://127.0.0.1:60946/PYdNFJKhAkk=/
For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".

Application finished.
```
</details>

**Comment by jmagman** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!

Thanks for your contribution.
